docs/tutorial.rst

```diff
@@ -704,64 +849,122 @@ not ancestral to these census nodes.
     >>> nodes = [i.id for i in ts.nodes() if i.flags==msprime.NODE_IS_CEN_EVENT]
     >>> ts_anc = ts.simplify(samples=nodes)
 
+.. _sec_advanced_features:
+
+Advanced features
+*****************
 
 ******************
-Recombination maps
+Historical samples
```

Comment:
maybe Historical samples should go under Demography?  They go naturally with "census events".

Reply:
Hmm, my thinking was that Demography should be specifically for objects within the `DemographicEvent` superclass, and objects that are modified by Demographic Events ie. `PopulationConfiguration` objects. I'd be inclined to keep it that way as these objects have a slightly different API compared with other inputs to `simulate()`.

Perhaps it would help if, somewhere in the tutorial, there is an example demonstrating the similarities and difference between historical sampling and census events? I could also make historical sampling the very first subsection in the 'Advanced Features' so that there's some continuity from one to the other?

Reply:
Oh, that makes sense. But I agree that these are easily confused at first, so something discussing the differences would be helpful.